neo/rawio/intanrawio.py

```diff
@@ -200,7 +201,9 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop, strea
             sl0 = i_start % block_size
             sl1 = sl0 + (i_stop - i_start)
 
-        sigs_chunk = np.zeros((i_stop - i_start, len(channel_ids)), dtype="uint16")
```

Comment:
So the original idea for this was that timestamps aren't not accessible so the only originally real data were all `uint16` (except temperature which I think was just ignored). So it is technically faster to just do it this way. I guess the question becomes do we want to allow access to timestamps? If not then this PR is really just in prep for #1402 once we have to worry about different dtypes for the amplifier data.

Reply:
As you mention, this is fixing the dtype for the temperature as well. So, it is fixing a present bug. 

If you mean computationally faster, I don't think there is any meaningful difference between getting the dtype from the data and assume uint16.

Concerning the timestamps. I don't see why they should not be made available as a separate stream. But I think that should be done in a different PR.

Reply:
I agree. It's a theoretical speedup rather than a practical/meaningful speedup.